test/unit/gateways/shift4_test.rb

```diff
@@ -223,6 +223,21 @@ def test_successful_refund
     assert_equal response.message, 'Transaction successful'
   end
 
+  def test_successful_credit
+    response = stub_comms do
+      @gateway.refund(@amount, @credit_card, @options.merge!(invoice: '4666309473', expiration_date: '1235'))
+    end.check_request do |_endpoint, data, _headers|
+      request = JSON.parse(data)
+      assert_equal request['card']['present'], 'N'
+      assert_equal request['card']['expirationDate'], '0924'
+      assert_nil request['card']['entryMode']
+      assert_nil request['customer']
+    end.respond_with(successful_refund_response)
+
+    assert response.success?
```

Comment:
💬 Comment:

I know that other tests have that check, but testing the response for success and its message is just unnecesary the mock is always going to return the success response.

Reply:
Understood! Thanks!
Change added!